settings.lua

```diff
@@ -264,6 +278,7 @@ function courseplay:switchCourseplayerSide(combine)
 		else
 			combine.cp.forcedSide = nil;
 		end;
+		combine:setCpVar('stopAtEnd', combine.cp.stopAtEnd, courseplay.isClient);
```

Comment:
FYI, all these legacy settings will at some point be converted to a SettingsList or derived object. Those (at the end of settings.lua) should probably have the MP updates built in later.

Reply:
Good to know, but all of there changes are probably going away anyway. They were just a result of me having huge desync issues, caused by other things, which are mostly fixed already.

Reply:
All the advanced config settings are SettingsList and derived classes so if a sync of those needed, probably that would be the place to implement it.